### PR TITLE
#652 - Verify install for pypi

### PR DESCRIPTION
**Pull request by genryxy**
Part of #652 
Verify package installation in pypiITCase

### CODE REVIEW

File: src/test/java/com/artipie/pypi/PypiITCase.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            .withCommand("tail", "-f", "/dev/null")
+            .withWorkingDirectory("/home/")
+            .withFileSystemBind(this.tmp.toString(), "/home");
+        this.cntn.start();
+    }
+
+    @Test
+    void pypiInstall() throws Exception {
+        final String url = String.format(
+            "http://host.testcontainers.internal:%d/my-pypi/", this.port
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/pypi/PypiITCase.java:97]: @genryxy I'd suggest to inline this variable and extract host `host.testcontainers.internal` into constant as it's used twice here and will be used in the future tests

File: src/test/java/com/artipie/pypi/PypiITCase.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                        .add("type", "fs")
+                        .add("path", this.tmp.resolve("repos").toString())
+                        .build()
+                )
+                .build()
+        ).build().toString();
+    }
+
+    private void addPackageToStorage() {
+        new TestResource("pypi-repo/alarmtime-0.1.5.tar.gz")
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/pypi/PypiITCase.java:137]: @genryxy this method has just one expression, I'd suggest to inline it

**Review by olenagerasimova — APPROVED** [review]
@genryxy added two comments, please, take a look

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
oops, wrong buttong

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks
@olegmoz looks good to merge

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#662` is not assigned, can't get performer

**Pull request merged**